### ISSUE TITLE
Make numbers be numbers in JSON data

### ISSUE THREAD

**Issue by stephenhouser** [stephenhouser/node-red-contrib-flexradio#23]
I would like the numbers reported back in messages, status, and other responses to actually be numbers and not numbers in strings so that I can save parsing data external to the nodes.

**Comment by stephenhouser** [comment]
How important is this in JavaScript/NodeRed? "==" comparisons will succeed, "===" will not.

**Comment by stephenhouser** [comment]
This change would be a problem because of inconsistency in the Flex protocol.

```
C7|stream create type=dax_rx dax_channel=1
S3D63A6E7|slice 0 dax=1 dax_clients=0
S3D63A6E7|stream 0x04000008 type=dax_rx dax_channel=1 slice=A client_handle=0x3D63A6E7 ip=192.168.10.20
S3D63A6E7|stream 0x04000008 type=dax_rx dax_channel=1 slice=A client_handle=0x3D63A6E7 ip=192.168.10.20
R7|0|4000008
C8|audio stream 0x4000008 slice 0 gain 50
```

**Comment by stephenhouser** [comment]
Fixed somewhere around commit 19fc2cf19e700e769981cdb59469a11f8ab6ea9d
